en/deploy/README.md

```diff
@@ -49,6 +59,8 @@ And save it as `.gitignore` in the top-level "djangogirls" folder.
 
 > **Note** The dot at the beginning of the file name is important!  If you're having any difficulty creating it (Macs don't like you to create files that begin with a dot via the Finder, for example), then use the "Save As" feature in your editor, it's bulletproof.
 
+> **Note** Did you notice that "db.sqlite3" was ignored?  That file is where we keep all of our posts.  Why would we skip it?  Your local system is your test setup and it is configured to use SQLite (`mysite/settings.py`).  Your application host, PythonAnywhere, will use something else.  In the case of PythonAnywhere, they use a more robust database called MySQL which holds more records and can deal with a lot more site visitors than SQLite can handle.  SQLite is perfect for test and development.  **All of the posts and users in your test environment are __NOT__ copied to PythonAnywhere, which leaves the deployed application feeling empty at first.  You will have to create users and posts just like you did in your test environment.**  Remember that your posts and user accounts are data.  Data is treated separately from your program.  Your test database is a great place to keep a copy of data that broke your system in the past as a way to make sure bugs stay closed days and months later as you improve your application.
+
```

Comment:
I definitely like your version better. 
One technical question though since I help someone with this on chat the other day.  I don't see where in the tutorial we changed the database settings to use MySQL.  I just followed along when I did it and I have a db.sqlite3 file in my PythonAnywhere instance.  I looked at the database section of PythonAnywhere and it says that I have no (MySQL) databases.  During the workshops, do you deliberately connect to MySQL and check your work or do you follow the tutorial pretty closely?  
I'll rework this section once I'm a little less confused.  
Thanks for the feedback.